offchain/danaswap-api/src/DanaSwap/Api.purs

```diff
@@ -67,7 +164,7 @@ initProtocol = do
   logDebug' "protocol init complete"
   pure
     { configUtxo
-    , poolVal
+    , poolAdrVal
     , liquidityMP
     , poolIdMP
     }
```

Comment:
Can we get docs on all the functions added?

Reply:
initProtocol wasn't actually from this pr but I threw a comment on it :man_shrugging: